2024/political-donations-by-party/index.html

```diff
@@ -0,0 +1,26 @@
+<html lang="en">
+    <head>
+        <title>Donations by Party</title>
+        <!-- PLACE ALL REPOSITORY CSS FILES IN A BRACKET LIKE THIS -->
+        <!-- build:css inline -->
+        <link rel="stylesheet" href="../../static/main.css" />
+        <!-- /build -->
+        <script src="http://d3js.org/d3.v4.js"></script>
+    </head>
+
+    <body>
+        <div id="grid-container">
+            <div id="grid-chart"></div>
+        </div>
+        <!-- EXTERNAL JS FILES -->
+        <script
+            src="https://cdn.jsdelivr.net/npm/d3plus@2.0.3"
+            type="text/javascript"
+        ></script>
```

Comment:
You're not using this import, yeah? You can delete it and the `<!-- EXTERNAL JS FILES -->` comment.